pkg/agent/plugin/svidstore/v1unofficial.go

```diff
@@ -0,0 +1,67 @@
+package svidstore
```

Comment:
This file, and the accompanying _test.go file are copied from the v1.go file, with import paths and type/function names changed to add the "unofficial" name.